Sources/App/main.swift

```diff
@@ -1,13 +1,20 @@
-import Vapor
+import AppLogic
```

Comment:
I thought about this too, I think `Executable` is extra clear. What do you think about that?